packages/magento-cart-items/Api/CartItem.graphql

```diff
@@ -24,6 +24,9 @@ fragment CartItem on CartItemInterface @injectable {
     price {
       ...Money
     }
+    price_including_tax {
+      ...Money
+    }
```

Comment:
price_including_tax is only available from Magento 2.4.5 onwards, but since 2.4.4 is [supported until April of 2025](https://experienceleague.adobe.com/docs/commerce-operations/release/planning/lifecycle-policy.html?lang=en) we want to keep supporting that version.

To build in this backwards compatibility we should add the schema here that defines these fields which  https://github.com/graphcommerce-org/graphcommerce/blob/canary/packages/magento-graphql/schema/Magento243to245.graphqls

Something like this:

```
extend type CartItemPrices {
    price_including_tax:  Money
}
```